### DISCUSSION TITLE
applying align-items inherit value

### DISCUSSION THREAD

**Discussion by herzaso** [tailwindlabs/tailwindcss#14554] [Help]
I have a use case in which I need to use align-items `inherit` but it seems that this is not supported by Tailwind. Also, I couldn't find how to configure the theme to support it (or use arbitrary value).

Any help will be appreciated 🙏

**Comment by wongjn** [comment]
You should be able to use [an arbitrary property class](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) like `[align-items:inherit]`.

**Comment by herzaso** [comment]
Thanks @wongjn that does work (I previously naively tried `items-[inherit]`).
Any reason why inherit / unset is not natively supported? Also, can I somehow change the theme to support that?

**Reply by wongjn** [reply]
> Any reason why inherit / unset is not natively supported? 

[Arbitrary value syntax](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values) is only available for ["dynamic-value" utilities](https://tailwindcss.com/docs/plugins#dynamic-utilities). The `items-`-related class names [are](https://github.com/tailwindlabs/tailwindcss/blob/ed3c5356b77b6c428e9c0ea5bc32719dcdf94128/src/corePlugins.js#L1327-L1333) [static utilities](https://tailwindcss.com/docs/plugins#static-utilities).

> Also, can I somehow change the theme to support that?

You could [disable the `alignItems` core plugin](https://tailwindcss.com/docs/configuration#core-plugins) and then [rewrite the utility classes as dynamic utilities](https://tailwindcss.com/docs/plugins#dynamic-utilities).

**Comment by oscartrevio** [comment]
You could use the `addUtilities` plugin to create `items-inherit` without the need to disable any core plugins or unnecessary complications. Here is the [link](https://tailwindcss.com/docs/plugins#overview) to the docs.

Your config file would look something like this:

```
plugins: [
    plugin(({ addUtilities }) => {
      addUtilities({
        ".items-inherit": {
          alignItems: "inherit",
        },
      });
    }),
  ],
```
